scripts/superdesk/editor/editor.js

```diff
@@ -602,18 +609,59 @@ function EditorService(spellcheck, $rootScope, $timeout, $q) {
     };
 
     /**
+     * Replace abbreviations.
+     * @param {Scope} scope
+     */
+    function replaceAbbreviations (scope) {
```

Comment:
I'm not sure if it should run `render` 2x during `commitScope`, it would make it slow imo, will test this